components/brave_rewards/browser/rewards_service_impl.cc

```diff
@@ -2200,8 +2200,19 @@ bool SavePendingContributionOnFileTaskRunner(PublisherInfoDatabase* backend,
   return backend->InsertPendingContribution(list);
 }
 
+void RewardsServiceImpl::OnCallBackPendingContribution(double amount) {
+  for (auto& observer : observers_)
+    observer.OnPendingContributionsTotal(this, amount);
+}
+
 void RewardsServiceImpl::OnSavePendingContribution(bool result) {
```

Comment:
as a side note results shouldn't be `bool` they should use result codes so we can provide additional info

Reply:
database return just true or false, so not sure what additional info we can get

Reply:
I think it's still better for consistency to just use Result everywhere in rewards service

Reply:
and something in the backend could change to potentially respond with more detailed errors